packages/terra-graphs/tests/wdio/Graphs/BarGraph/BasicBarGraphs/NegativeBars-spec.js

```diff
@@ -0,0 +1,12 @@
+Terra.describeViewports('Negative bars', ['tiny', 'medium', 'large'], () => {
+  describe('Validates', () => {
+    before(() => browser.url('/raw/tests/terra-graphs/graphs/bar-graph/basic-bar-graphs/negative-bars'));
```

Comment:
Since all of [these](https://github.com/cerner/terra-graphs/tree/automated-test-stable-branch/packages/terra-graphs/tests/wdio/Graphs/BarGraph/BasicBarGraphs) are tests for BarGraph, we should consolidate these different examples into one single test spec/file and just load the url for each example to minimize the number of test files instead of having a separate test file for each example. Here is an [example](https://github.com/cerner/terra-core/blob/main/packages/terra-button/tests/wdio/button-spec.js) for testing multiple scenarios all in a single test spec.

Reply:
As suggested wrote all the tests for each graph in a single [file](https://github.com/cerner/terra-graphs/pull/106/commits/8f1b7d01eb1f006f648f922b7b0ca25095b52fb2#diff-9ca661f8bd98f35e239b477f1bf147440a31d0659491d029cd783cce3d866006R2).